RtspServer/MediaTypes/RtpVideoSink.cs

```diff
@@ -197,20 +199,21 @@ internal override void SendPackets()
 
                         transportContext.RtpTimestamp += ClockRate * 1000;
 
-                        frame.Timestamp = (int)transportContext.RtpTimestamp;
+                        frame.Timestamp = transportContext.RtpTimestamp;
 
                         //Fire a frame changed event manually
-                        if (RtpClient.FrameChangedEventsEnabled) RtpClient.OnRtpFrameChanged(frame, transportContext, true);
+                        if (RtpClient.FrameChangedEventsEnabled)
+                            RtpClient.OnRtpFrameChanged(frame, transportContext, true);
 
                         //Todo, should not copy packets
 
-                        //Take all the packet from the frame                            
+                        //Take all the packet from the frame
                         IEnumerable<Rtp.RtpPacket> packets = frame;
 
                         //Clear the frame to reset sequence numbers (could add method to do this)
                         //frame.RemoveAllPackets();
 
-                        if (Loop) frame = new Rtp.RtpFrame();
+                        if (Loop) frame = [];
```

Comment:
Lovely how it just looks like a plain array but its actually newing up a RtpFrame 👍